proxy/internal/modules/store/zipstore.go

```diff
@@ -1,7 +1,13 @@
 package store
 
 import (
+	"database/sql"
+
+	"oss.indeed.com/go/modprox/pkg/config"
 	"oss.indeed.com/go/modprox/pkg/coordinates"
+	database "oss.indeed.com/go/modprox/pkg/db"
```

Comment:
rule of thumb: if you have to use a package alias, the package name should be improved;
in this case maybe just name the package `/pkg/database`?

Reply:
the package was already named `database`, but I needed to rename the directory to match the package name